src/Stripe.net/Services/Invoices/StripeInvoiceService.cs

```diff
@@ -50,10 +50,15 @@ public virtual StripeInvoice Update(string invoiceId, StripeInvoiceUpdateOptions
         }
 
         public virtual StripeInvoice Pay(string invoiceId, StripeRequestOptions requestOptions = null)
+        {
+            return this.Pay(invoiceId, null, requestOptions);
+        }
+
+        public virtual StripeInvoice Pay(string invoiceId, StripeInvoicePayOptions payOptions, StripeRequestOptions requestOptions = null)
```

Comment:
I did this to avoid a breaking change but maybe it'd be cleaner to force the option?